lib/fluent/plugin/in_forward.rb

```diff
@@ -293,6 +298,9 @@ def on_message(msg, chunk_size, conn)
         log.warn "Input chunk size is larger than 'chunk_size_warn_limit':", tag: tag, host: conn.remote_host, limit: @chunk_size_warn_limit, size: chunk_size
       end
 
+      tag = @tag.dup if @tag
+      tag = "#{@add_tag_prefix}.#{tag}" if @add_tag_prefix
```

Comment:
Is it ok without checking tag is an empty string?

Reply:
Ah, good catch. I will add empty check on `configure`.